<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHM</class>
 <widget class="QWidget" name="IHM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Formulaire de saisie</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>447</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="lName">
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
     <widget class="QLabel" name="lBirthdate">
      <property name="text">
       <string>Date de naissance :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="lFirstname">
      <property name="text">
       <string>Pr√©nom :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lPrintFirstname">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lPrintName">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lPrintBirthDate">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leName">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leFirstname">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>21</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCalendarWidget" name="calendarWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leName</sender>
   <signal>editingFinished()</signal>
   <receiver>lPrintName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onEditFirstname(QString)</slot>
  <slot>onEditName(QString)</slot>
 </slots>
</ui>
